site/frontend/src/pages/compare/compile/compile-page.vue

```diff
@@ -230,12 +274,22 @@ const filteredSummary = computed(() => computeSummary(comparisons.value));
   />
   <OverallSummary :summary="filteredSummary" />
   <Aggregations :cases="comparisons" />
+  <div class="warning" v-if="selfCompareBackend">
+    Comparing Cranelift against LLVM baseline!
```

Comment:
Maybe something like

```suggestion
    Note: comparing results of the baseline LLVM backend to the cranelift backend.
```